src/main/resources/localstack/s3/clients/clients.json

```diff
@@ -50,7 +50,8 @@
     "contact": "Publisher",
     "created": 1609459200,
     "roles": [
-      "GENERATOR"
+      "GENERATOR",
+      "SHARER"
```

Comment:
Do we need this? Publishers generally won't have SHARER to start with

Reply:
As I understand it, any participant type can have the SHARER permission - is there some reason this won't work? Afaik once these changes go live, a publisher would be able to include another publisher in their sharing list.

Reply:
Yes it will work, but it's a very uncommon scenario. What I'm getting at is that we should keep a GENERATOR-only example in this file, as that's a much more common case